src/app/auth/login.component.ts

```diff
@@ -40,7 +41,7 @@ export class LoginComponent implements OnInit {
     const domainFromQueryParams = this.route.snapshot.queryParams['domain'];
     this.store
       .pipe(
-        select(configSelectors.get('defaultDomain')),
+        select(getDefaultDomain),
```

Comment:
do we need to test that?

Reply:
This is just an extracted constant, no need to change tests